draft-ietf-quic-recovery.md

```diff
@@ -338,27 +338,41 @@ mean variation.
 The calculation of smoothed_rtt uses path latency after adjusting RTT samples
 for acknowledgement delays. These delays are computed using the ACK Delay
 field of the ACK frame as described in Section 19.3 of {{QUIC-TRANSPORT}}.
-For packets sent in the ApplicationData packet number space, a peer limits
-any delay in sending an acknowledgement for an ack-eliciting packet to no
-greater than the value it advertised in the max_ack_delay transport parameter.
-Consequently, when a peer reports an Ack Delay that is greater than its
-max_ack_delay, the delay is attributed to reasons out of the peer's control,
-such as scheduler latency at the peer or loss of previous ACK frames.  Any
-delays beyond the peer's max_ack_delay are therefore considered effectively
-part of path delay and incorporated into the smoothed_rtt estimate.
+
+When 0-RTT or 1-RTT ack-eliciting packets are received, a peer does not delay
+acknowledging them any longer than the period it advertised in the max_ack_delay
+transport parameter; see Section 18.2 of {{QUIC-TRANSPORT}}.
```

Comment:
We need to add: ...except during the handshake, if packets cannot be processed when received due to lack of packet protection keys.